crates/stages/src/stages/sender_recovery.rs

```diff
@@ -101,6 +101,11 @@ impl<DB: Database> Stage<DB> for SenderRecoveryStage {
         // Chunks are submitted instead of individual transactions to reduce the overhead of work
         // stealing in the threadpool workers.
         let chunk_size = self.commit_threshold as usize / rayon::current_num_threads();
+        // prevents an edge case
+        // where the chunk size is either 0 or too small
```

Comment:
how it's possible that we get zero if `self.commit_threshold  > 0`?

Reply:
right, and we set the threshold to 10 in `execute_intermediate_commit`. Makes sense!